src/components/dialog/dialog.js

```diff
@@ -608,6 +605,11 @@ function MdDialogProvider($$interimElementProvider) {
      */
     function showBackdrop(scope, element, options) {
 
+      if (options.disableParentScroll) {
+        // !! DO this before creating the backdrop
```

Comment:
Can you elaborate this comment? I.e., `Restore the scroll position before creating the backdrop because...`